grafana-plugin/src/containers/MobileAppConnection/MobileAppConnection.tsx

```diff
@@ -192,12 +192,19 @@ export const MobileAppConnection = observer(({ userPk }: Props) => {
     );
   }
 
+  const isMobile = window.matchMedia('(max-width: 768px)').matches;
+
   return (
     <VerticalGroup>
       <div className={cx('container')}>
         <Block shadowed bordered withBackground className={cx('container__box')}>
           <DownloadIcons />
         </Block>
+        {QRCodeValue && isMobile && (
```

Comment:
maybe better to have a check of `QRCodeDataParsed && isMobile` here instead? `QRCodeDataParsed` has `any` type 🙃 if it's `undefined` (which can happen if `JSON.parse` in `getParsedQRCodeValue` fails), then we will get "Cannot read property ... of undefined" error here

Reply:
I took inspiration from line 169, maybe we should change it there too?

Reply:
There situation is a bit different because we have first part that depends just on `QRCodeValue` and then second nested part that depends on `QRCodeDataParsed`. Second part already have a check for existence of `QRCodeDataParsed`.
![image](https://github.com/grafana/oncall/assets/12073649/3baea300-abc4-4880-ab44-de1ce3859925)

So I think we should be good to not touch this place and only use `QRCodeDataParsed && isMobile` condition in your new change

Reply:
ah ok cool, updated